lib/ain-evm/src/core.rs

```diff
@@ -159,9 +159,9 @@ pub struct ValidateTxInfo {
 }
 
 pub struct TransferDomainTxInfo {
-    pub from: String,
-    pub to: String,
-    pub native_address: String,
+    pub from: [u8; 20],
```

Comment:
Can we type this ? 
Something like `pub type XAddress = [u8; 20];`

Reply:
Sounds good. Will add this in

Reply:
Switched in c6ff318